bin/rails

```diff
@@ -1,4 +1,10 @@
 #!/usr/bin/env ruby
+if ENV['RAILS_ENV'] == 'test'
+  require 'simplecov'
+  SimpleCov.start 'rails'
+  puts "required simplecov"
```

Comment:
Probably just advertising their :gem: ! We can remove it later if it isn't adding value :moneybag: